provider/schema/execution.go

```diff
@@ -316,7 +334,28 @@ func (e *ExecutionData) resolveResourceValues(ctx context.Context, meta ClientMe
 			err = fmt.Errorf("recovered from panic: %s", r)
 		}
 	}()
-	if err = e.resolveColumns(ctx, meta, resource, resource.table.Columns); err != nil {
+
+	cols := e.Db.Dialect().Columns(resource.table)
```

Comment:
Maybe. `cq_fetch_date` still needs to be added before `cq_id` (because `cq_id` is generated using a hash of PKs, which `cq_fetch_date` is) and it might be cryptic (to always keep cq_id after the PKs are set? or we could make it a rule...)

Also internal columns are resolved after `PostRowResolver` call fof some reason (same in the old code with SDKColumns resolving) and I wanted to keep that, if that's not important we can discuss.

Reply:
cq_id shouldn't be generated from cq_fetch_date thats a big logical issue, you want your cq_id to be consistent over time.

Reply:
cq_id issue fixed...  other loopy code still there, unless calling `PostRowResolver` before SDKColumns isn't a thing any more.